qiskit/providers/ibmq/api_v2/clients/auth.py

```diff
@@ -112,7 +117,7 @@ def user_hubs(self):
         """
         response = self.client_api.hubs()
 
-        hubs = []
+        hubs = []  # type: Hubs
```

Comment:
Let's not introduce type hints _inside_ the functions during this round - as with other comments, we can reevaluate once we have a wider picture (don't worry as well it is produces warnings, etc).

Reply:
It was the warnings that made me decide to add the type hints for the variables. But, that makes sense. I'll focus on the bigger picture as we go on for now. I removed them in the most recent PR.